daemon/logger/splunk/splunk.go

```diff
@@ -0,0 +1,256 @@
+// +build linux
```

Comment:
@LK4D4 yes, that is right, you just need to have access to the Splunk instance, which can be on Linux.